src/DDoSX/DomainClient.php

```diff
@@ -67,6 +68,32 @@ public function create(Domain $domain)
             });
     }
 
+    /**
+     * @param Domain $domain
+     * @return null
+     */
+    public function deploy(Domain $domain)
+    {
+        $this->post('v1/domains/' . $domain->name . '/deploy');
+
+        return null;
+    }
+
+    /**
+     * @param $domainName
+     * @return Ip
+     */
+    public function getIpByDomainName($domainName)
```

Comment:
As this is on the domain class, do we need the `ByDomainName` part of this function's name?

Reply:
updated the name